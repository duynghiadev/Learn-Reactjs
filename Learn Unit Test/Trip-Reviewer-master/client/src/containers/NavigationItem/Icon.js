import styled from 'styled-components';
import FontAwesome from 'react-fontawesome';

const Icon = styled(FontAwesome)`
  color: white;
  width: 20px;
  text-align: center;
`;

export default Icon;
