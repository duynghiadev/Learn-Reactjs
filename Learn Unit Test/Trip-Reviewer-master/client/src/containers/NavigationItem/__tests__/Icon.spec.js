import React from 'react';
import { create } from 'react-test-renderer';

import Icon from '../Icon';

const mockComponent = props => <Icon {...props} />;

describe('NavigationItem/<Icon />', () => {
  it('should render component', () => {
    const tree = create(mockComponent()).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
