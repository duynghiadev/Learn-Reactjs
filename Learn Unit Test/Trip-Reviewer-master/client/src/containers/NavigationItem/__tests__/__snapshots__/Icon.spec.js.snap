// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavigationItem/<Icon /> should render component 1`] = `
<span
  aria-hidden={true}
  className="fa fa-undefined Icon-cxuQhR gTgVNd"
/>
`;
